vcd/sample_vcd_test_config.json

```diff
@@ -120,11 +120,14 @@
 
     "//": "Leave the network names, the second storage profile, and the media name empty to skip their creation",
     "//": "They are not needed for terraform-provider-vcd testing, but are useful for go-vcloud-director",
+    "//": "Likewise, leaving the user or password empty will skip the org user creation",
```

Comment:
Comment became like a book right now (long). It's better to move it just above the respective lines ("orgUser" one), and make it more declarative, like "Leave the user or password empty will skip the org user creation".

Reply:
done